### DISCUSSION TITLE
how to change the image src instead of using base64 i want to use image file name only in converted json?

### DISCUSSION THREAD

**Discussion by senthil177cse** [GrapesJS/grapesjs#4354] [Q&A]
Hi @artf ..
how to change the image src instead of using base64 i want to use image file name only in converted json?
![image](https://user-images.githubusercontent.com/50009624/170936188-82b0ffe2-34c5-49c6-9fbc-a8790c33b5e4.png)

**Comment by artf** [comment]
Be sure to have the asset manager uploader properly configured https://grapesjs.com/docs/modules/Assets.html#uploading-assets
